services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -313,6 +319,7 @@ private void installEgressFlow(InstallEgressFlow command) throws SwitchOperation
     private void doProcessTransitFlow(final CommandMessage message, String replyToTopic, Destination replyDestination)
             throws FlowCommandException {
         InstallTransitFlow command = (InstallTransitFlow) message.getData();
+        logger.info("Processing transit flow. Switch {}", command.getSwitchId());
```

Comment:
need flow_id here

PS For ingress and egress rules we have used term "install".